combat-trainer.lic

```diff
@@ -171,13 +176,27 @@ class SetupProcess
       game_state.prepare_summoned_weapon(last_summoned)
     else
       @equipment_manager.wield_weapon(game_state.weapon_name, game_state.weapon_skill)
+      if game_state.whirlwind_trainables && game_state.is_whirlwind_trainable?(game_state.weapon_skill)
```

Comment:
What do you think about having `is_whirlwind_trainable?` return false if `whirlwind_trainables` is empty? Then you only need the second clause in this `if`.